src/pybind/mgr/prometheus/module.py

```diff
@@ -1578,6 +1578,21 @@ def get_pool_repaired_objects(self) -> None:
             self.metrics[path].set(stats['stat_sum']['num_objects_repaired'],
                                    labelvalues=(stats['poolid'],))
 
+    def get_all_daemon_health_metrics(self) -> None:
+        daemon_metrics = self.get_daemon_health_metrics()
+        self.log.debug('metrics jeje %s' % (daemon_metrics))
+        for daemon_name, health_metrics in daemon_metrics.items():
+            for health_metric in health_metrics:
+                path = f'daemon_health_metrics{daemon_name}{health_metric["type"]}'
+                self.metrics[path] = Metric(
+                    'counter',
```

Comment:
I wonder if this metric should be a gauge instead of a counter?  The "Health metrics for Ceph daemons" description is super opaque, but, judging by the rest of the PR, the primary use case is `SLOW_OPS` count and that number can go both up and down.